Newbranch

Tchapis opened the pull request:
No conflicts and merging newbranch into main.

Commit:
change on newbranch

Commit:
Merge branch 'main' into newbranch

Tchapis reviewed (commented):
I approve this pull request. <insert Obama meme giving himself a medal>

ser316asu reviewed (changes requested):
remove the merge conflicts  and create a new pull request and raise an appeal for approval
you should get rid of this
<<<<<<< and =======

Tchapis committed:
Update README.md

Removed <<<<<<<<, >>>>>>>>> and ======== from the file. It's weird it told me there were no merge conflicts.

ser316asu commented:
Approve this PR
